jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/Asn1EncodedDataRouter.java

```diff
@@ -353,6 +324,59 @@ public void processEncodedTimUnsecured(ServiceRequest request, JSONObject consum
       logger.info("TIM deposit response {}", responseList);
    }
 
+   public String signTIM(String encodedTIM, JSONObject consumedObj) {
+      logger.debug("Sending message for signature! ");
+      String base64EncodedTim = CodecUtils.toBase64(
+            CodecUtils.fromHex(encodedTIM));
+      JSONObject metadataObjs = consumedObj.getJSONObject(AppContext.METADATA_STRING);
+      // get max duration time and convert from minutes to milliseconds (unsigned
+      // integer valid 0 to 2^32-1 in units of
+      // milliseconds.) from metadata
+      int maxDurationTime = Integer.valueOf(metadataObjs.get("maxDurationTime").toString()) * 60 * 1000;
+      String timpacketID = metadataObjs.getString("odePacketID");
+      String timStartDateTime = metadataObjs.getString("odeTimStartDateTime");
+      String signedResponse = asn1CommandManager.sendForSignature(base64EncodedTim,maxDurationTime);
+      try {
+         String hexEncodedTim = CodecUtils.toHex(
+               CodecUtils.fromBase64(
+                     JsonUtils.toJSONObject(JsonUtils.toJSONObject(signedResponse).getString("result")).getString("message-signed")));
+
+         JSONObject TimWithExpiration = new JSONObject();
```

Comment:
This variable should start with a lowercase letter to be consistent with other variable declarations in this class.

Reply:
Updated this to start with a lowercase letter